### ISSUE TITLE
Code for visualizing the Effective Receptive Fields (ERFs)

### ISSUE THREAD

**Issue by Min-Sheng** [hangg7/deformable-kernels#4]
Hi,

Thank you for your great work.
Could you release the code for visualizing the Effective Receptive Fields (ERFs) ?

Thanks!

**Comment by mosquitobite** [comment]
Yes, it would be nice of you if you can release the code or give some hints for visualizing!

**Comment by ArthurZucker** [comment]
Hi, 
Awesome work thank you very much!
I am also wondering if you have any plans of releasing the Effective Receptive Field visualization code? 
There's pretty much none out there... 
Hints or clues as to how to compute it would also be of great help.
Thanks again :)
